Makefile

```diff
@@ -22,6 +22,11 @@ MD ?= @
 # TLS related configuration
 XI_BSP_TLS ?= wolfssl
 
+#detect if the build happen on Travis
+ifdef TRAVIS_OS_NAME
```

Comment:
Because this had to be repeated everywhere used and I wanted to centralise this test in order to make it easier to maintain.